### PR TITLE
[CODE CAMP] AbstractSampleView

### PR DESCRIPTION
**Pull request by marcus-oscarsson**
Hi,

Updated AbstractSampleView and related classes, mainly integrates the Shape handling into SampleVIew. This makes the SampleView handling more similair between the two version, the Shapes object was removed.

This effectively makes #544 unnecessary 

All the best,
Marcus

### CODE REVIEW

File: HardwareObjects/SampleView.py [hunk 1]
```diff
@@ -29,11 +33,511 @@
...
+
+        if shape:
+            shape.de_select()
+
+    def is_selected(self, sid):
+        """
+        Check if Shape with <sid> is selected.
+
+        Returns:
+            (Boolean) True if Shape with <sid> is selected False otherwise
```
Comment (rhfogh) [inline HardwareObjects/SampleView.py:169]: If there is no shape with that sid it will currently return None. Is taht OK? Should hte documentation say so?

Comment (marcus-oscarsson) [reply HardwareObjects/SampleView.py:169]: Probably better if it returns False

**Comment by marcus-oscarsson** [comment]
@all, @IvarsKarpics It would be great it we could get this merged our at least a comment if there is something fundamentally wrong. Since this solves and issue described in #544 that prevents MXCuBE3 from running properly

**Comment by marcus-oscarsson** [comment]
Thanks :)

**Pull request merged**